baybe/searchspace/core.py

```diff
@@ -303,23 +304,69 @@ def estimate_product_space_size(parameters: Iterable[Parameter]) -> MemorySize:
 
     def transform(
         self,
-        data: pd.DataFrame,
+        df: pd.DataFrame | None = None,
+        /,
+        *,
+        allow_missing: bool = False,
+        allow_extra: bool | None = None,
+        data: pd.DataFrame | None = None,
     ) -> pd.DataFrame:
-        """Transform data from experimental to computational representation.
-
-        This function can e.g. be used to transform data obtained from measurements.
-        Continuous parameters are not transformed but included.
+        """Transform parameters from experimental to computational representation.
 
         Args:
-            data: The data to be transformed. Must contain all specified parameters, can
-                contain more columns.
+            df: The dataframe to be transformed. The allowed columns of the dataframe
+                are dictated by the `allow_missing` and `allow_extra` flags.
```

Comment:
Don't we need double backticks here?

Reply:
The good old question 😄 will check in the rendered doc